### DISCUSSION TITLE
TextInput autoCapitalize prop not working

### DISCUSSION THREAD

**Discussion by danielbettech** [mantinedev/mantine#2221] [Q&A] [answered]
I'm trying to autoCapitalize the text inside a `<TextInput />` component as the user types.

```
 <TextInput
    label='Name'
    autoCapitalize
    {...customerForm.getInputProps('name')}
  />
```

I found the autoCapitalize props in the intellisens but it doesn't work. Any help?

**Comment by rtivital** [comment]
TextInput does not implement any extra logic for this prop, it is added directly to `input` element – if it does not work with `input` element, then it will not work with TextInput as well.

**Comment by danielbettech** [comment, marked as answer]
Yes its not working with autoCapitalize, I resolve the issue like this.

```
 <TextInput
    label='Name'
    styles={{ input: { textTransform: 'uppercase' } }}
    {...customerForm.getInputProps('name')}
  />
```

Thanks!

**Reply by danielbettech** [reply]
I notice that with this approach the value in state its still in lowercase.
